taxi/forms.py

```diff
@@ -0,0 +1,30 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number",)
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
```

Comment:
The `DriverLicenseUpdateForm` should also validate the `license_number` to ensure it meets the criteria of 3 uppercase letters followed by 5 digits. Implementing this validation is crucial to fulfill the task requirements.